nixos/modules/services/networking/nebula.nix

```
@@ -185,7 +201,15 @@ in
             outbound = netCfg.firewall.outbound;
           };
         } netCfg.settings;
-        configFile = format.generate "nebula-config-${netName}.yml" settings;
+        configFile = format.generate "nebula-config-${netName}.yml" (
+          warnIf
+            ((settings.lighthouse.am_lighthouse || settings.relay.am_relay) && settings.listen.port < 1)
+            ''
+              Nebula network '${netName}' is configured as a lighthouse or relay, and its port is ${builtins.toString settings.listen.port}.
+              You will experience connectivity issues.
+            ''
```

Would be helpful to include a link to the documentation about this: https://nebula.defined.net/docs/config/listen/#listenport



The connectivity issues are for a `port` of 0, so `< 1` is odd. Can `types.port` even be negative?

```suggestion
            ''
              Nebula network '${netName}' is configured as a lighthouse or relay, and its port is ${builtins.toString settings.listen.port}.
              You will likely experience connectivity issues: https://nebula.defined.net/docs/config/listen/#listenport
            ''
```